app/views/components/docs/document-list.yml

```diff
@@ -0,0 +1,50 @@
+name: Document list
+description: An ordered list of links to documents including document type and when updated.
+body: |
+  Accepts a group of documents from a content item and outputs a list of links to those documents. The component expects the public_updated_at value as a date object, but can also convert a correctly formatted date string.
+
+  If a group_index parameter is included the component adds link tracking attributes to each link.
+
+  Documents are presented in an ordered list as the component expects that the ordering of the documents is relevant.
+accessibility_criteria: |
+  The component must:
+
+  * inform the user how many items are in the list
+
+  Links in the component must:
+
+  * accept focus
+  * be focusable with a keyboard
+  * be usable with a keyboard
+  * indicate when it has focus
+  * Change in appearance when touched (in the touch-down state)
+  * Change in appearance when hovered
+  * Usable with touch
+  * Usable with speech
+  * Have visible text
+fixtures:
+  default:
+    items:
+    - public_updated_at: 'Mon, 27 Jun 2016 10:29:44 BST +01:00'
```

Comment:
If you remove the quotes Rails should make this a date automagically:
https://stackoverflow.com/questions/2656196/ruby-how-do-i-define-a-datetime-field-in-yaml